casa/OS/Path.cc

```diff
@@ -395,23 +395,12 @@ String Path::makeAbsoluteName (const String& inString) const
 	return inString;
     }
     // Otherwise we have a relative pathname.
-    // Remove a possible leading . or ./
+    // Remove a possible leading ./
```

Comment:
I would reword this since it partly describes the code but not completely.
No real suggestion, but removing this line would be fine by me. The line above would be enough for me.

Reply:
Changed as suggested.